runtime/onert/api/CMakeLists.txt

```diff
@@ -22,6 +22,11 @@ endif (ANDROID)
 target_include_directories(${ONERT_DEV} PUBLIC include)
 set_target_properties(${ONERT_DEV} PROPERTIES PUBLIC_HEADER "${NNFW_API_HEADERS}")
 
+if(CMAKE_BUILD_TYPE STREQUAL "release")
```

Comment:
It needs to handle upper/lower case. (ex. "Release", "RELEASE", etc)

Reply:
Our build process takes type strings other than "release" ?